File: openapi-client/cpp-tiny/lib/Models/EmailAddressesTypes.cpp


#include "EmailAddressesTypes.h"

using namespace Tiny;

EmailAddressesTypes::EmailAddressesTypes()
{
}

EmailAddressesTypes::EmailAddressesTypes(std::string jsonString)
{
	this->fromJson(jsonString);
}

EmailAddressesTypes::~EmailAddressesTypes()
{

}

void
EmailAddressesTypes::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);


}

bourne::json
EmailAddressesTypes::toJson()
{
    bourne::json object = bourne::json::object();


    return object;

}

File: swagger-client/dart/lib/model/email_addresses_types.dart
part of swagger.api;

class EmailAddressesTypes {
  
  EmailAddressesTypes();

  @override
  String toString() {
    return 'EmailAddressesTypes[]';
  }

  EmailAddressesTypes.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
  }

  Map<String, dynamic> toJson() {
    return {
     };
  }

  static List<EmailAddressesTypes> listFromJson(List<dynamic> json) {
    return json == null ? new List<EmailAddressesTypes>() : json.map((value) => new EmailAddressesTypes.fromJson(value)).toList();
  }

  static Map<String, EmailAddressesTypes> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, EmailAddressesTypes>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new EmailAddressesTypes.fromJson(value));
    }
    return map;
  }
}
